src/spikeinterface/core/basesorting.py

```diff
@@ -529,6 +535,9 @@ def to_spike_vector(self, concatenated=True, extremum_channel_inds=None, use_cac
             spike_dtype = spike_dtype + [("channel_index", "int64")]
             ext_channel_inds = np.array([extremum_channel_inds[unit_id] for unit_id in self.unit_ids])
 
+        if use_cache and self._cached_spike_vector is None:
+            self._cache_spike_vector_from_parent()
```

Comment:
I would call this  self._custum_cache_spike_vector() because it can cache this from something else than the parent. No ?

Reply:
Do you have an example in mind?

Reply:
Maybe a `MergeUnitsSorting`? In this case it's from multiple parents. Custom sounds ok to me